kratos/integration/hexahedron_gauss_legendre_integration_points.h

```diff
@@ -45,10 +45,12 @@ class KRATOS_API(KRATOS_CORE) HexahedronGaussLegendreIntegrationPoints1
         return 1;
     }
 
-    static IntegrationPointsArrayType& IntegrationPoints()
+    static const IntegrationPointsArrayType IntegrationPoints()
```

Comment:
You may return the const reference instead of by value as it is a static variable

Reply:
Done